sci-electronics/kicad-packages3d/kicad-packages3d-5.1.4.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs cmake-utils
+
+DESCRIPTION="Electronic Schematic and PCB design tools 3D package libraries"
+HOMEPAGE="https://kicad.github.io/packages3d/"
+SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="CC-BY-SA-4.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="occ +oce"
+
+REQUIRED_USE="|| ( occ oce )"
+
+DEPEND=""
+RDEPEND=">=sci-electronics/kicad-5.1.0[occ=,oce=]"
+
+CHECKREQS_DISK_BUILD="9G"
```

Comment:
`tmpfs            10G   10G  4.0K 100% /tmp`
```

FAILED: CMakeFiles/install.util 
cd /tmp/portage/sci-electronics/kicad-packages3d-5.1.4/work/kicad-packages3d-5.1.4_build && /usr/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
/usr/lib/portage/python3.6/isolated-functions.sh: line 270: echo: write error: No space left on device
```
*sigh*

Reply:
I also expect you to figure the correct amount for this, /tmp/portage/sci-electronics/kicad-packages3d-5.1.4/ was ~9,6 GB before it died

Reply:
Oh, I did not notice since my tmpfs is 16GB. Since portage reports the final size of the build directory and also has the disk requirements set, it could just report it as a QA violation. I'll see if I have time to put up a PR for that.

I'll look at the other LD issues, thanks for the pointer.